README.md

```diff
@@ -11,6 +12,36 @@
 
 This project is under active development and at the moment provides no useful resources. [Internal notes on the project can be seen in the GitLab Wiki](https://git.un.org/undrr/web-backlog/-/wikis/Mangrove:-the-UNDRR-Component-library).
 
+## Quick Start
+
+### Installation
```

Comment:
Suggest instead of adding this here we direct users to https://unisdr.github.io/undrr-mangrove/?path=/docs/getting-started-a-getting-started-guide--docs and ensure this use case is captured either on that page or one of the related pages